_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -148,33 +152,80 @@ commit
 end
 ```
 
-Finally, let's check connectivity between boxes:   
-
+Checking connectivity between boxes:
 ```
-RP/0/RP0/CPU0:ios#ping 10.1.1.20
+RP/0/RP0/CPU0:ios#ping 10.1.1.10
 Mon May  9 08:36:33.071 UTC
 Type escape sequence to abort.
-Sending 5, 100-byte ICMP Echos to 10.1.1.20, timeout is 2 seconds:
+Sending 5, 100-byte ICMP Echos to 10.1.1.10, timeout is 2 seconds:
 !!!!!
 Success rate is 100 percent (5/5), round-trip min/avg/max = 1/5/20 ms
 RP/0/RP0/CPU0:ios#
 ```
 
-Let's copy public part of key from Ubuntu box and allow access without password. For IOS-XR image 6.1.x commands below:   
 
+Let's copy public part of key from **Ubuntu** box and allow access without password. At first we should connect to Ubuntu and copy file to XR via SCP
 ```
-vagrant port
+vagrant ssh ubuntu
+scp -P 57722 /home/vagrant/.ssh/id_rsa.pub  vagrant@<XR's interface address>:/home/vagrant/id_rsa_ubuntu.pub
 
-scp -P <port used for XR(guest) port 57722> /path/to/host/public_key/id_rsa.pub vagrant@localhost:/home/vagrant/
+```
 
+Next step to apply copied key inside XR:
+```
 vagrant ssh xr
+cat /home/vagrant/id_rsa_ubuntu.pub >> /home/vagrant/.ssh/authorized_keys
+```
+
+
+If we want paswordless SSH from Ubuntu to XR CLI, issue the following commands on XR:
+
+```
+scp vagrant@10.1.1.10:/home/vagrant/.ssh/id_rsa_pub.b64 /disk0:/id_rsa_pub.b64
```

Comment:
Because the above command needs to be executed in XR linux shell, bundle it with the previous scp command. See my comment on line 170.